### PR TITLE
Pam s3 read urls

### PR DESCRIPTION
**Pull request by tosan88**
See https://github.com/Financial-Times/publish-availability-monitor/pull/101

Currently in testing

*NOTE*: Etcd key should be set before merging this

### CODE REVIEW

File: publish-availability-monitor@.service [hunk 1]
```diff
@@ -22,7 +22,7 @@ ExecStart=/bin/bash -c '\
     --env="KAFKA_PROXY_HOST=kafka-rest-proxy" \
     --env="ETCD_PEERS=http://%H:2379" \
     --env "QUEUE_ADDR=http://%H:8080" \
-    --env "S3_URL=http://$BUCKET.s3.amazonaws.com/" \
+    --env "S3_URL=/" \
```
Comment (BerniVarga) [inline publish-availability-monitor@.service]: Can't we get rid of this env variable?

Comment (BerniVarga) [inline publish-availability-monitor@.service]: Did we add a clean-up task in the backlog to cover this one?

Comment (tosan88) [inline publish-availability-monitor@.service]: No, I can add one ;)

**Comment by tosan88** [comment]
I close this and let a new release to include these changes as per discussed with @yolandeleungFT

**Pull request closed without merging**